v2.1/sql-faqs.md

```diff
@@ -39,6 +39,30 @@ For example, this is how you’d use `RETURNING` to return an auto-generated [`S
 > INSERT INTO users (name) VALUES ('mike') RETURNING id;
 ~~~
 
+## What is transaction contention?
+
+Transaction contention occurs when transactions issued from multiple
+clients at the same time operate on the same data.
+
+This can cause transactions to wait on each other and decrease
```

Comment:
This doesn't need to be its own paragraph. Please make it the second sentence of the previous paragraph.